src/components/mdx/CheatSheet.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+export default function CheatSheet({title, children}) {
+  return (
+    <div className="bg-white shadow-md p-6 border-gray-300 sm:px-8 border border-gray-100 rounded-md m-6">
+      <details>
+        <summary className="font-medium text-2xl text-extrabold text-gray-900">
```

Comment:
**praise:** I like seeing these elements :)
**thought:** Is it necessary to hide cheatsheets from the user? I thought it would make a good `<Spoiler>` component.

Reply:
I don’t think it’s necessary at all. They love the “roll ups” in Notion which is where this was migrated from. Personally I’d love to see a better treatment of the cheat sheets and don’t see any practical benefit to hiding them.